clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10394,7 +10394,8 @@ static void getOtherSPIRVTransOpts(Compilation &C,
       ",+SPV_INTEL_fpga_argument_interfaces"
       ",+SPV_INTEL_fpga_invocation_pipelining_attributes"
       ",+SPV_INTEL_fpga_latency_control"
-      ",+SPV_INTEL_task_sequence";
+      ",+SPV_INTEL_task_sequence"
+      ",+SPV_INTEL_bindless_images";
```

Comment:
Could you also update `clang/test/Driver/sycl-spirv-ext.c` to cover your new entry?

Reply:
Oh, I missed that. Will do.

Reply:
Done.